Gems/PhysX/Code/Tests/CharacterGameplayControllerTests.cpp

```diff
@@ -98,6 +98,14 @@ namespace PhysX
             }
         }
 
+        /**
+        * Function to delete the ground for tests where the ground can interfere with the intent of the test. 
+        */
+        void DeleteGround()
```

Comment:
nit: Potentially prefer 'Destroy' or 'Remove' ground (as 'delete' has different connotations in C++)

Reply:
Updated to RemoveGround

Reply:
Thank you! 🙂